sdk/data/azappconfig/policy_sync_token.go

```diff
@@ -26,7 +26,7 @@ type syncTokenPolicy struct {
 }
 
 func newSyncTokenPolicy() *syncTokenPolicy {
-	return &syncTokenPolicy{}
+	return &syncTokenPolicy{syncTokens: map[string]syncToken{}}
```

Comment:
Is it intended the map will always be empty?

Reply:
We initialize it to an empty map when the policy is created.